osgp/protocol-adapter-oslp/web-device-simulator/src/main/java/org/opensmartgridplatform/webdevicesimulator/service/OslpChannelHandler.java

```diff
@@ -494,6 +495,21 @@ private void sleep(final Long sleepTime) {
             }
         }
 
+        response = this.handleRequest(request, response, device);
+
+        // Update device
+        device.setSequenceNumber(expectedSequenceNumber);
+        this.deviceManagementService.updateDevice(device);
+
+        // Write log entry for response
+        LOGGER.debug("Responding: {}", response);
+
+        return response;
+    }
+
+    @SuppressWarnings("squid:MethodCyclomaticComplexity")
```

Comment:
according to https://github.com/SmartSocietyServices/Way-of-Working-Wiki/wiki/Suppression-of-SonarQube-warnings-like-FalsePositives a justification why the specific warning is being suppressed should be given

Reply:
Added a comment.